samples/multicore/hello_world/README.rst

```diff
@@ -26,18 +25,54 @@ This sample shows how to inform the build system about dedicated sources for add
 The sample comes with the following additional files:
 
 * :file:`Kconfig.sysbuild` - This file is used to add Sysbuild configuration that is passed to all the images.
-  ``SB_CONFIG`` is the prefix for sysbuild’s Kconfig options.
+  ``SB_CONFIG`` is the prefix for sysbuild's Kconfig options.
 * :file:`sysbuild.cmake` - The CMake file adds additional images using the :c:macro:`ExternalZephyrProject_Add` macro.
   You can also add the dependencies for the images if required.
 
 Both the application and remote cores use the same :file:`main.c` that prints the name of the DK on which the application is programmed.
 
 Building and running
 ********************
+
 .. |sample path| replace:: :file:`samples/multicore/hello_world`
 
 .. include:: /includes/build_and_run_sb.txt
 
+The remote board needs to be specified using ``SB_CONFIG_REMOTE_BOARD``.
+As shown below, it is recommended to use configuration setups from :file:`sample.yaml` using the ``-T`` option to build the sample.
+
+**nRF5340 DK**
+
+You can build the sample for application and network cores as follows:
+
+.. code-block:: console
+
+   west build -p -b nrf5340dk_nrf5340_cpuapp -T sample.multicore.hello_world.nrf5340dk_cpuapp_cpunet .
+
+**nRF54H20 PDK**
+
+You can build the sample for application and radio cores as follows:
+
+.. code-block:: console
+
+   west build -p -b nrf54h20pdk_nrf54h20_cpuapp -T sample.multicore.hello_world.nrf54h20pdk_cpuapp_cpurad .
+
+You can build the sample for application and PPR cores as follows:
+
+.. code-block:: console
+
+   west build -p -b nrf54h20pdk_nrf54h20_cpuapp -T sample.multicore.hello_world.nrf54h20pdk_cpuapp_cpuppr .
+
+Note that :ref:`zephyr:nordic-ppr` is used in the configuration above to automatically launch PPR core from the application core.
+
+An additional configuration setup is provided to execute code from RAM on the PPR core.
+This configuration uses :ref:`zephyr:nordic-ppr-ram` and disables :kconfig:option:`CONFIG_XIP` on the PPR core.
+It can be built as follows:
+
+.. code-block:: console
+
+   west build -p -b nrf54h20pdk_nrf54h20_cpuapp -T sample.multicore.hello_world.nrf54h20pdk_cpuapp_cpuppr_ram .
```

Comment:
```suggestion
nRF54H20 PDK
  You can build the sample for application and radio cores as follows:

  .. code-block:: console

     west build -p -b nrf54h20pdk_nrf54h20_cpuapp -T sample.multicore.hello_world.nrf54h20pdk_cpuapp_cpurad

  You can build the sample for application and PPR cores as follows:

  .. code-block:: console

     west build -p -b nrf54h20pdk_nrf54h20_cpuapp -T sample.multicore.hello_world.nrf54h20pdk_cpuapp_cpuppr

  Note that :ref:`zephyr:nordic-ppr` is used in the configuration above to automatically launch PPR core from the application core.

  An additional configuration setup is provided to execute code from RAM on the PPR core.
  This configuration uses :ref:`zephyr:nordic-ppr-ram` and disables :kconfig:option:`CONFIG_XIP` on the PPR core.
  It can be built as follows:

  .. code-block:: console

     west build -p -b nrf54h20pdk_nrf54h20_cpuapp -T sample.multicore.hello_world.nrf54h20pdk_cpuapp_cpuppr_ram
```